network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/store/NetworkProtectionRepository.kt

```diff
@@ -36,6 +36,7 @@ interface NetworkProtectionRepository {
     var enabledTimeInMillis: Long
     var serverDetails: ServerDetails?
     var clientInterface: ClientInterface?
+    var disabledDueToAccessRevoked: Boolean
```

Comment:
As discussed, maybe better to rename to `vpnAccessRevoked` as it VPN enable/disable doesn't related to whether the VPN entitlement is revoked or not (eg. VPN can be ON even if access is revoked)